presto-native-execution/presto_cpp/main/PrestoExchangeSource.cpp

```diff
@@ -171,6 +181,8 @@ void PrestoExchangeSource::processDataResponse(
           } while (curr != start);
           PrestoExchangeSource::updateMemoryUsage(-freedBytes);
         });
+    REPORT_ADD_HISTOGRAM_VALUE(
```

Comment:
Do we also want to report 0 when page is empty?

Reply:
FYI: page can't be empty here.

Reply:
Yeah I mean when empty is true. If so we need to move the report line outside of the if condition.